sources/packages/web/src/views/student/StudentProfileCreate.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <student-page-container>
+    <template #header>
+      <v-row class="text-center">
```

Comment:
do we need `row and col` for this content? shouldn't `div` be enough?

Reply:
No need, I will change it as per suggestion.

Reply:
Changed.